app/presenters/search_results_presenter.rb

```diff
@@ -159,4 +177,8 @@ def next_page_number
   def previous_page_number
     current_page_number - 1
   end
+
+  def is_scoped_to_manual?
```

Comment:
I could be missing something but what ties this method to manuals as oppose to an generic scope? As far as I can tell I think that the contents of this method could easily be moved to the `is_scoped?` method on line 130 and this method removed as it's all generic now?